hank-client/src/main/java/com/liveramp/hank/config/EnvironmentValue.java

```diff
@@ -0,0 +1,20 @@
+package com.liveramp.hank.config;
+
+public class EnvironmentValue {
```

Comment:
So lean the other way.  I think advantages to even trivial getter methods are
 
-  it's easier to add debug info to trace access when it's a method
-  refactoring is far easier -- if you want to change the return value to be a modified version of the field (or even just wrap it in an immutable collection, etc), you don't have to trace through all the usages to do it
- refactoring again, it's easier to turn it into an interface later without rewriting consumer code.

In general in a good encapsulated design I don't think the consumer needs to know what's going on behind the scenes of the getX.  Obviously plenty of arguments either way, but I don't really agree with clean code on this.